packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx

```diff
@@ -188,8 +188,16 @@ export class CommandBar extends BaseComponent<ICommandBarProps, ICommandBarState
     const className = css(
       isLink ? ('ms-CommandBarItem-link ' + styles.itemLink) : ('ms-CommandBarItem-text ' + styles.itemText),
       !item.name && ('ms-CommandBarItem--noName ' + styles.itemLinkIsNoName),
-      (expandedMenuItemKey === item.key) && ('is-expanded ' + styles.itemLinkIsExpanded)
+      (expandedMenuItemKey === item.key) && ('is-expanded ' + styles.itemLinkIsExpanded),
+      item.inactive ? styles.inactive : ''
     );
+
+    let tooltipContent = '';
+
+    if (item.title) {
```

Comment:
Doesn't this add a tooltip to every command bar item? Is that intended?

Reply:
@micahgodbolt only if they provide a title attribute

Reply:
ah title attribute, yup. gotcha. was thinking title was the text in the button.